pkg/okteto/kubetoken/cache.go

```diff
@@ -0,0 +1,122 @@
+package kubetoken
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/okteto/okteto/pkg/log"
```

Comment:
Let's use the same name as in the rest of the code for consistency
```suggestion
	oktetoLog "github.com/okteto/okteto/pkg/log"
```